bust-a-move-framework/src/test/java/bustamove/game/CannonTest.java

```diff
@@ -43,8 +44,9 @@ public void fireMaxRighttAngle() {
         for (int i = 0; i < 60; i++) {
             cannon.stepUp();
         }
+        Bubble loaded = game.getBubbles().get(1);
```

Comment:
Bubble.loadNextBubble() still contains .add(...) instead of addFirst(...). That might explain the weird ordering